src/index.html

```diff
@@ -22,6 +22,384 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="https://imgbb.com/"
+        class="logo"
+      >
+        <img
+          src="https://i.ibb.co/rQYtvhT/moyo-1.png"
+          alt="moyo logo"
+          class="logo__img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="index.html"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="product">
+      <article
+        class="product-card"
+        data-qa="card"
+      >
```

Comment:
Two attributes can be in one line
```suggestion
      <article class="product-card" data-qa="card">
```

Reply:
Still not fixed. It will be better if you fix it 
<img width="896" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/c34663f0-4537-46f0-9af1-b376147e753d">